doc/user/load_save.md

```diff
@@ -604,11 +607,80 @@ mlpack::data::ResizeImages(images, info, 160, 160);
 // contains the names of the resized images.
 std::vector<std::string> smSheeps =
     {"sm_sheep_1.jpg", "sm_sheep_2.jpg", "sm_sheep_3.jpg", "sm_sheep_4.jpg",
-     "sm_sheep_5.jpg", "sm_sheep_6.jpg"};
+     "sm_sheep_5.jpg", "sm_sheep_6.jpg", "re_sheep_7.jpg", "re_sheep_8.jpg",
+     "re_sheep_9.jpg"};
```

Comment:
10 images is a lot for the example, do we really need all of them?  Also I think maybe these in particular should be `sm_sheep` not `re_sheep`.

Reply:
Yes, because they have different dimensions covering different cases, this is the reason behind them